packages/ui-components/src/components/ProfileButton.js

```diff
@@ -117,14 +113,16 @@ export const ProfileButton = ({ user, MenuOptions, className }) => {
         <StyledButton
           onClick={event => setAnchorEl(anchorEl ? null : event.currentTarget)}
           className={className}
-          endIcon={<Avatar>{userInitial}</Avatar>}
+          endIcon={<StyledAvatar initial={userInitial}>{userInitial}</StyledAvatar>}
```

Comment:
No `src` here?

Reply:
I didn't put it in because I preferred how the initial looked. But on reflection, I think that is silly and the user can decide.